cobc/scanner.l

```diff
@@ -304,6 +310,21 @@ static void *	copy_literal (cb_tree l);
 	}
 }
 
+<*>^{AREA_A}[ ]*/"." {
+	count_lines (yytext + 9); /* skip "\n#area_a\n" */
+	if (cobc_in_procedure) {
+		(void) cb_verify (cb_non_word_in_area_a,
```

Comment:
This is definitely mis-leading. From the standard:

> 5.2.7    Punctuation
The separators comma and semicolon may be used anywhere the separator space is used in general formats and other  syntactic specifications. In the compilation group, these separators are interchangeable.
The separator period, when specified in a general format, is required when that format is used.

Therefore we don't use "punctuation" in any msgid.
From the definitions in `PROCEDURE DIVISION`:

> A paragraph consists of a paragraph-name followed by a separator period and by zero, one, or more successive sentences or, if the paragraph-name is omitted, one or more successive sentences following the procedure division header or a section 
header.

So when we want to reference a _new_ procedure-name with a missing separator period (not here) before or after, we can name it `procedure-name` or, if we know what we have "paragraph header" / "section header".

The main question: Is there any use of explicit raising an error / warning with a separator period here?